sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/xml/xpath.scala

```diff
@@ -56,8 +56,16 @@ abstract class XPathExtract extends BinaryExpression with ExpectsInputTypes with
 }
 
 @ExpressionDescription(
-  usage = "_FUNC_(xml, xpath) - Evaluates a boolean xpath expression.",
-  extended = "> SELECT _FUNC_('<a><b>1</b></a>','a/b');\ntrue")
+  usage = "_FUNC_(xml, xpath) - Evaluates a boolean an XPath expression.",
```

Comment:
This is wrong, right? In Hive, the document is like:

`Returns true if the XPath expression evaluates to true, or if a matching node is found.`